azure-eventgrid/src/test/java/com/microsoft/azure/eventgrid/EventGridTests.java

```diff
@@ -1,10 +1,16 @@
 package com.microsoft.azure.eventgrid;
 
-import com.microsoft.azure.eventgrid.implementation.EventGridClientImpl;
-import com.microsoft.azure.eventgrid.models.EventGridEvent;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.joda.JodaModule;
+import com.microsoft.azure.eventgrid.implementation.*;
+import com.microsoft.azure.eventgrid.models.*;
```

Comment:
Please remove all imports with .*. It should be easily configured in the IDE settings.